src/configure/configure.ts

```diff
@@ -432,16 +432,18 @@ class PipelineConfigurer {
 
     private async getAzureResourceDetails(): Promise<void> {
         // show available subscriptions and get the chosen one
-        let subscriptionList = extensionVariables.azureAccountExtensionApi.filters.map((subscriptionObject) => {
+        let subscriptionList = extensionVariables.azureAccountExtensionApi.subscriptions.map((subscriptionObject) => {
             return <QuickPickItemWithData>{
-                label: <string>subscriptionObject.subscription.displayName,
+                label: `${<string>subscriptionObject.subscription.displayName} (${<string>subscriptionObject.subscription.subscriptionId})`,
```

Comment:
Why do we want to show subscription id ?

Reply:
Two subscriptions can have same name. When I was testing there were two MSDN subscription with exact same name - becomes very confusing.

Reply:
In that case we can add this to the either description or details of the QuickPickItem, that display's it in less bold manner

Reply:
Changed to description